### ISSUE TITLE
back button/gesture closes app

### ISSUE THREAD

**Issue by VivienJustJ** [elliottetzkorn/t4t#138]
When in a top level function of the app: feed, messages, reactions, profile, and you use the phone's back function, the app closes. 

Could it simply revert to the previous function? For instance, if you're scrolling the feed, the l then toggle to reactions, then go back, have it return to the feed? 

Not sure about *most* users, but that feels intuitive.

**Comment by elliottetzkorn** [comment]
Isn't this how all apps work? The tab isn't a new page, you are still on the homepage, so a back click from there makes sense to me that it would close the app.

**Comment by VivienJustJ** [comment]
nope. on instagram, for instance, when you go "back" from the messaging section, it takes you to the feed. same with the tumblr app. 

when you open the apps they uniformly take you to your feed first, and only going back from the feed closes the app.
